lib/OpenQA/Command.pm

```diff
@@ -107,4 +108,16 @@ sub url_for ($self, $path) {
     return Mojo::URL->new($self->apibase . $path)->to_abs(Mojo::URL->new($self->host));
 }
 
+sub retry_tx ($self, $client, $tx, $handle_args, $retries = undef, $delay = undef) {
+    $delay //= $ENV{OPENQA_CLI_RETRY_SLEEP_TIME_S} // 3;
+    $retries //= $ENV{OPENQA_CLI_RETRIES} // 0;
+    for (;; --$retries) {
```

Comment:
That would probably be easier to read as a while loop.

Reply:
I have no strong feelings about this though, if nobody else cares just keep the fancy for loop.